server/rpc.go

```diff
@@ -313,7 +313,9 @@ func coreSegMetadata(segData *net.SegData) (*core.SegTranscodingMetadata, error)
 	}
 	var err error
 	profiles := []ffmpeg.VideoProfile{}
-	if len(segData.FullProfiles2) > 0 {
+	if len(segData.FullProfiles3) > 0 {
+		profiles, err = makeFfmpegVideoProfiles(segData.FullProfiles3)
```

Comment:
Needs a test case for this, probably in [`TestCoreSegMetadata_FullProfiles`](https://github.com/livepeer/go-livepeer/blob/d9ea3358e0a43f07a0fd78880f37bd48750c1be2/server/segment_rpc_test.go#L330)